sstables/compaction.cc

```diff
@@ -562,14 +589,16 @@ class compaction {
         };
     }
 
+    virtual void on_new_partition() = 0;
+
     virtual shared_sstable create_new_sstable() const = 0;
 
     // select a sstable writer based on decorated key.
```

Comment:
minor: comment should be updated

Reply:
Will force push with the comment change. thanks